libc/src/errno/CMakeLists.txt

```diff
@@ -1,9 +1,20 @@
+# If we are in full build mode, we will provide the errno definition ourselves,
+# and if we are in overlay mode, we will just re-use the system's errno.
+# We are passing LIBC_FULL_BUILD flag in full build mode so that the
+# implementation of libc_errno will know if we are in full build mode or not.
+set(full_build_flag "")
```

Comment:
Wondering if this shouldn't be top-level like the stuff in `__support/macros` and set similar to other config options.

Reply:
I think @michaelrj-google also has some usage for full build mode detection inside the implementations, so I will probably move this to `_get_common_compile_options` later.

Reply:
Alright, put a TODO here then.

Reply:
Done.